loris/resolver.py

```diff
@@ -60,12 +60,20 @@ class SimpleFSResolver(_AbstractResolver):
 
     def __init__(self, config):
         super(SimpleFSResolver, self).__init__(config)
-        self.cache_root = self.config['src_img_root']
+        if 'src_img_roots' in self.config:
+            self.cache_roots = self.config['src_img_roots']
+        else:
+            self.cache_roots = [self.config['src_img_root']]
 
-    def is_resolvable(self, ident):
+    def get_fp(self, ident):
         ident = unquote(ident)
-        fp = join(self.cache_root, ident)
-        return exists(fp)
+        for directory in self.cache_roots:
```

Comment:
:ok: I may refactor later (not a big deal) and I think the fact that we're using `exists()` vs. `os.exists()`, which would add an extra function lookup in every iteration of the loop is probably a good start.